package.json

```diff
@@ -108,7 +105,6 @@
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.4.2",
     "@embroider/test-setup": "^0.44.1",
-    "@fullhuman/postcss-purgecss": "^4.0.3",
```

Comment:
We want to keep this for tailwind, I think. If it is not currently working, we should work on making sure we get purging working with tailwind

Reply:
+1 made some tests locally and purging don't work at the moment, i reverted the commit in this PR and will try to tackle this "issue" in another PR, have something "basic" working where the extra CSS is purged for this repo and for repo using this addon but it will for sure require more testing and some refactor (like purging only when we build in production)